scripts/xtensa-build-zephyr.sh

```diff
@@ -39,11 +39,17 @@ Outputs in \$(west topdir)/build-*/ directories
 usage: $0 [options] [ platform(s) ] [ -- cmake arguments ]
 
        -a Build all platforms.
+
        -j n Set number of make build jobs for rimage. Jobs=#cores by default.
            Ignored by "west build".
+
```

Comment:
will these two empty lines be printed with `-h`? Not sure they're useful / helpful?

Reply:
Yes the newlines between each option are in the -h output, I found the earlier, packed output hard to read. I can remove them if you feel strongly about it (I don't)